Gemfile

```diff
@@ -15,4 +15,6 @@ group :test do
   gem 'therubyracer', '~> 0.12.0'
   gem 'cldr-plurals-runtime-rb', '~> 1.0.0'
   gem 'cldr-plurals-runtime-js', '~> 1.0.0'
+
+  gem 'libv8', '= 3.16.14.11' # 2016-04-20 lock to older working version on Clang/OS X (3.16.14.13 won't compile)
```

Comment:
Sorry about this. I'm not too happy with having to include libv8 or therubyracer at all, since they never seem to install without a huge fight. Perhaps it's time to depend on execjs and let the developer decide what runtime to use.